source/plugin/basics/items.rst

```diff
@@ -0,0 +1,61 @@
+==================
+Working with Items
+==================
+
+Items are identified through an ``ItemStack``. An ``ItemStack`` is an inventory item with information such as the amount of the item in the stack, the type of the item, and extra data such as durability.
+
+Checking an Item's Type
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Checking the type of the item is very simple. You just need to call the ``getItem()`` method in the ``ItemStack``.
+
+.. code-block:: java
+
+    import org.spongepowered.api.item.ItemType;
+    import org.spongepowered.api.item.ItemTypes;
+    import org.spongepowered.api.item.inventory.ItemStack;
+    
+    public boolean isStick(ItemStack stack) {
+        ItemType type = stack.getItem();
+        return type == ItemTypes.STICK;
+    }
+    
+See how simple that is? Now we know that our item is a stick, but sticks can stack, so how many of them are there?
```

Comment:
"Now we know that our item is a stick, but sticks can stack, so how many of them are there?"

Run-on sentence - maybe "Because sticks can stack, we can also find out how many are present"